draft-ietf-rats-corim.md

```diff
@@ -1591,129 +1591,118 @@ The Verifier SHOULD set the `authorized-by` field in Accepted Claims Set entries
 to a format which contains only a key, for example the `tagged-cose-key-type`
 format. Using a common format makes it easier to compare the field.
 
-## Accepted Claims Set extension using CoMID tags or triples
+## Accepted Claims Set extension using CoMID triples
 
-In the Accepted Claims Set extension phase, a CoRIM Appraisal Context and
-an Evidence Appraisal Policy are used by the Verifier to find CoMID tags or triples which
-match the Attester. Tags/triples which match are accepted, and the Accepted Claims Set
-is extended using Endorsements etc. from the accepted tags.
+In the Accepted Claims Set extension phase, a CoRIM Appraisal Context and an Evidence Appraisal Policy are used by the Verifier to find CoMID tags or triples which match the ACS.
+Tags/triples which match are accepted, and the Accepted Claims Set is extended using Endorsements etc. from the accepted tags.
 
-[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/136
+Each triple is processed independently of other triples.
+If any triple within the CoRIM Appraisal Context does not match the Accepted Claims Set then ACS extensions in that triple are not processed.
+If a triple does not match the ACS then this does not affect whether the Verifier tries to match other triples against the ACS.
 
-### Comparing and processing CoMID tags or triples
+Note that the order of triples processing is important. Processing a triple may result in modifications to the ACS which allow other triples to match the modified ACS.
+The Verifier SHOULD ensure that it processes triples in the correct order (for example by processing all triples which match against a particular ACS entry or environment after that ACS entry has been modified).
```

Comment:
This is security-relevant, it can't be a SHOULD.  It has to be a well-defined MUST. 


```suggestion
The Verifier SHOULD ensure that it processes triples in the correct order (for example by processing all triples which match against a particular ACS entry or environment after that ACS entry has been modified).
```

Reply:
I think it is ok to make this MUST, but I expect the result of processing in the wrong order to be an attestation failure so this will fail safe.